website/src/content/development/page-content-layouts.mdx

```diff
@@ -0,0 +1,224 @@
+---
+title: Page Content Layouts
+navLevels: 2
+---
+
+At commercetools, one core design principle is a visual hierarchy, having a clean and usable user interface using spacings. As part of these guidelines, it's essential to properly structure a page's overall content.
```

Comment:
The original sentence was more correct. For instance, visual hierarchy is not a design principle but a heuristic. I would try to keep the terminology and meaning close to the one before.

How about this? ⬇️

```suggestion
At commercetools, one of our core design heuristics is about visual hierarchy, to have a clean and usable user interface by making use of spacings. As part of these guidelines, it's essential to properly structure the page's overall content.
```

Reply:
Hi @emmenko, thanks for pointing this out. I think the grammar and readability of the sentence are a bit off. How about
> At commercetools, visual hierarchy is one of our core design heuristics to create a clean and user-friendly interface by utilizing spacing. For visual hierarchy, it's crucial to structure the page content properly.

Reply:
Yes sounds good.

Reply:
@Anshuman71 Do you push the sentence update here?

Reply:
ℹ️ I pushed the suggestion now. https://github.com/commercetools/merchant-center-application-kit/pull/3057/commits/097fd23945f66b0b669ce754f1808ccaff5f187f